com.chartboost.helium/Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/HeliumUnityAdWrapper.kt

```diff
@@ -0,0 +1,225 @@
+package com.chartboost.heliumsdk.unity
+
+import android.app.Activity
+import android.graphics.Color
+import android.util.DisplayMetrics
+import android.util.Log
+import android.view.Gravity
+import android.view.View
+import android.widget.FrameLayout
+import android.widget.RelativeLayout
+import com.chartboost.heliumsdk.ad.HeliumAd
+import com.chartboost.heliumsdk.ad.HeliumBannerAd
+import com.chartboost.heliumsdk.ad.HeliumBannerAd.HeliumBannerSize
+import com.chartboost.heliumsdk.ad.HeliumFullscreenAd
+import com.chartboost.heliumsdk.ad.HeliumRewardedAd
+import com.unity3d.player.UnityPlayer
+
+class HeliumUnityAdWrapper(private val ad: HeliumAd) {
+    // This is the container for the banner.  We need a relative layout to position the banner in one of the 7
```

Comment:
if you use /** */, it will show up as javadocs as part of the IDE which is nice instead of just regular // comments. Also, this comment probably isn't meant for the activity.

Reply:
Thanks!